deps.edn

```diff
@@ -1,6 +1,7 @@
 {:deps {org.clojure/clojure {:mvn/version "1.11.1"},
-        org.apache.pdfbox/pdfbox {:mvn/version "2.0.29"}
-        org.apache.pdfbox/pdfbox-tools {:mvn/version "2.0.29"}}
+        org.apache.pdfbox/pdfbox {:mvn/version "3.0.1"}
+        org.apache.pdfbox/pdfbox-io {:mvn/version "3.0.1"}
```

Comment:
Why is this being required in here and not in `project.clj`?

Or better: does `project.clj` need this library too?

Reply:
I assumed that you have `project.clj` because you wanted to have both `lein` and `deps.edn` support.

Is that right?

Reply:
yes, the duplicate line in `project.clj` should have been this dependency